### DISCUSSION TITLE
SignUp form working but there's no user entry added to the profiles table

### DISCUSSION THREAD

**Discussion by anjanesh** [supabase/supabase#14486] [Questions]
My src\components\pages\Auth\SignUp.jsx seems to be working - I even get a valid JSON :

```
XHRPOST
https://somerandomid.supabase.co/auth/v1/signup
[HTTP/3 200 OK 84ms]

	
id	"xxxxxxxxxxxxxxx"
aud	"authenticated"
role	"authenticated"
email	"someone@gmail.com"
phone	""
confirmation_sent_at	"2023-05-20T11:11:29.421435963Z"
app_metadata	Object { provider: "email", providers: […] }
provider	"email"
providers	[ "email" ]
0	"email"
user_metadata	Object { }
identities	[]
created_at	"2023-05-20T11:11:29.421435963Z"
updated_at	"2023-05-20T11:11:29.421435963Z"
```

But I don't see the data in my `profiles` table which was created using the default user management profile starter.

```
create table profiles (
  id uuid references auth.users on delete cascade not null primary key,
  updated_at timestamp with time zone,
  username text unique,
  full_name text,
  avatar_url text,
  website text,

  constraint username_length check (char_length(username) >= 3)
);
```

**Comment by GaryAustin1** [comment]
What is your trigger and trigger function (part of user management starter) ?

**Comment by anjanesh** [comment]
Apologies, the user data from create user form **_is_** getting inserted into the table.

Its just that the table UI in supabase is showing one (now two) record but when double clicking on the record, a panel on the right hand shows up showing 3 records which is correct.

![image](https://github.com/supabase/supabase/assets/85973/3ff86add-24a9-4ac1-b378-3ce343bd9ebf)

![image](https://github.com/supabase/supabase/assets/85973/65d309dd-b938-4f07-bbd7-7cc858ad64d4)

Not sure why there aren't 3 records in the table editor in the default's display at https://app.supabase.com/project/xxx/editor/9999

**Reply by GaryAustin1** [reply]
Was a user already added before you added the user management and trigger?

**Reply by anjanesh** [reply]
I was following [this course](https://www.udemy.com/course/reactjs-with-supabase) and did a signup 3 times using the same browser session I believe. (Had to make some changes in the code esp src\context\AuthContext.jsx to get it to work properly)

**Reply by GaryAustin1** [reply]
Not much one can do to help as not clear what all you have run and when.
If it is adding them correctly now then I assume your first one was added before you had the trigger and function (which you have not shown) working correctly.  
I have no idea what is in that course, and how the react side of things could impact the database side and user management sql.

**Reply by anjanesh** [reply]
I found that all the data got inserted but the supabase was showing one one record in the table view editor and on double clicking the one record a right hand side popup showed all 3 records intact which were added by the react's signup form.
